packages/template-retail-react-app/app/pages/checkout/partials/pickup-address.jsx

```diff
@@ -0,0 +1,129 @@
+/*
+ * Copyright (c) 2025, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import React, {useState} from 'react'
+import {FormattedMessage, useIntl} from 'react-intl'
+
+// Components
+import {Box, Button, Container, Text} from '@salesforce/retail-react-app/app/components/shared/ui'
+import {
+    ToggleCard,
+    ToggleCardSummary
+} from '@salesforce/retail-react-app/app/components/toggle-card'
+import AddressDisplay from '@salesforce/retail-react-app/app/components/address-display'
+
+// Hooks
+import {useCheckout} from '@salesforce/retail-react-app/app/pages/checkout/util/checkout-context'
+import {useCurrentBasket} from '@salesforce/retail-react-app/app/hooks/use-current-basket'
+import {useShopperBasketsMutation, useStores} from '@salesforce/commerce-sdk-react'
+
+export default function PickupAddress() {
```

Comment:
```suggestion
const PickupAddress() => {
...
}
export default PickupAddress
```

So we're consistent.

Reply:
done